src/nomad_simulations/schema_packages/atoms_state.py

```diff
@@ -552,7 +552,16 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
                 )
 
 
-class AtomsState(Entity):
+class State(Entity):
```

Comment:
Didn't end up using this after all, but I left it for now in case we need it. But will remove before merge if not